.github/workflows/release.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - "pre*"
```

Comment:
Is it possible to be a bit more strict?

`preferences-updates`, `preconfiguration` or such branches will also match that rule, and this is not what we want.

Maybe an explicit `prerelease` branch?

Reply:
Ok, maybe a single branch is good enough. It's not like we should have multiple prerelease versions at the same time